app/models/vacancy.rb

```diff
@@ -66,7 +66,7 @@ class Vacancy < ApplicationRecord
   has_many :job_applications, dependent: :destroy
   has_one :equal_opportunities_report, dependent: :destroy
   has_many :organisation_vacancies, dependent: :destroy
-  has_many :organisations, through: :organisation_vacancies, dependent: :destroy, after_add: :refresh_geolocation, after_remove: :refresh_geolocation
+  has_many :organisations, through: :organisation_vacancies, dependent: :destroy, validate: false, after_add: :refresh_geolocation, after_remove: :refresh_geolocation
```

Comment:
looks good - but personally I would have added a test and/or comment to explain. It's hard to spot in this long line

Reply:
Added both @starswan